pkg/portutil/port_allocate_others.go

```diff
@@ -1,3 +1,4 @@
+//+build !linux
```

Comment:
https://github.com/containerd/nerdctl/blob/master/go.mod#L3

The Go 1.17 has been recommended version for the nerdctl project. FYI https://github.com/containerd/nerdctl/blob/master/go.mod#L3.

And in Go 1.17 `//go:build ` is the recommended way to specify the build target.

If you want to backport the build target to Go 1.16, you may need to add `//+build` tag to everywhere we use `//go:build` to maintain the code style. FYI https://github.com/containerd/nerdctl/search?q=repo%3Acontainerd%2Fnerdctl+%2F%2Fgo%3Abuild

BTW, I don't think this would be necessary